changelogs/fragments/482-radius_server-regression-test-hostname.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
Could be changed to 'trivial', since only documentation and test changes are being made.

Reply:
Done.